sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
I don't know the  best way to do it, but it needs to be accessible somehow (even if it's not super obvious). 

Maybe it should be a new section on the bottom of the splash page? Materialize's home page would be an example to follow.